oaknational/Oak-Web-Application
# LESQ-278 Feat/home page nav

## Description

- Add homePageNav to storybook
- fully functional and style

## Issue(s)

Fixes #

## How to test

1. Go to

https://deploy-preview-1898--owa-storybook.netlify.thenational.academy/?path=/story/navigation-homepagetabimagenav--home-page-tab-image-nav

2. Click on Navigation/HomePageNav
3. You should see _______

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
